android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 18 11:37:38 CEST 2018
+#Fri Oct 30 11:54:28 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
if you update the gradle wrapper, you should also update the wrapper itsself. Just call `./gradlew wrapper` and files like `gradlew.sh`, `gradlew.bat` and others will be replaced with newer versions. :-)